### PR TITLE
Launch CiV on demand

### PR DESCRIPTION
**Pull request by YadongQi**
Check environment variable ${ENABLE_CIV_BALLOON}, ballon guest if it is
enable.
Pasue CiV if no LG app launched;
Resume CiV if there are LG app to be launched;
Monitor all files/folders under ~/Android to ensure them with r/rx permission in Others group

Tracked-On: OAM-98178
Signed-off-by: Yadong Qi <yadong.qi@intel.com>

### CODE REVIEW

File: host/src/pack/dpkg/opt/cfc/mwc/bin/resume_civ.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        echo "QMP connected!"
+    else
+        echo "Failed to connect QMP!"
+        return -1
+    fi
+}
+
+function resume_civ()
+{
+    echo "Send: stop && toggle_instance_status.value=0"
```
Comment (lchen43) [inline host/src/pack/dpkg/opt/cfc/mwc/bin/resume_civ.sh:56]: echo "Send: stop && toggle_instance_status.value=0"  value should be 1

**Review by shuangwan01 — APPROVED** [review]
LGTM

**Review by shuangwan01 — APPROVED** [review]
LGTM

**Review by shuangwan01 — APPROVED** [review]
LGTM

**Review by renchenglei — APPROVED** [review]

**Review by renchenglei — APPROVED** [review]

**Review by renchenglei — APPROVED** [review]

**Review by renchenglei — APPROVED** [review]

File: host/src/pack/dpkg/opt/cfc/mwc/bin/lg_launcher.sh [hunk 2]
```diff
@@ -8,10 +8,14 @@ then
...
 fi
 
+# Resume CiV
+/opt/cfc/mwc/bin/resume_civ.sh
+
 num_start_app=`ps aux | grep startapp | grep -v grep | wc -l`
 
 if [ "$num_start_app" -lt "1" ];
 then
+    /opt/cfc/mwc/bin/resume_civ.sh
```
Comment (lchen43) [inline host/src/pack/dpkg/opt/cfc/mwc/bin/lg_launcher.sh:18]: partner feedback the change introduce boot time impact. may need optimize to reduce redundant check.

File: host/src/pack/dpkg/opt/cfc/mwc/bin/lg_launcher.sh [hunk 3]
```diff
@@ -8,10 +8,14 @@ then
...
     if [ "$(adb get-state | grep "device" | awk '{print $1}')" != "device" ];
     then
         /opt/lg/bin/startapp
```
Comment (lchen43) [inline host/src/pack/dpkg/opt/cfc/mwc/bin/lg_launcher.sh:21]: need a one line to trigger android media scan

**Pull request merged**